test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,58 @@
+require "test_helper"
+
+describe "RecipesController" do
+  describe "root" do
+    it "shows the root page" do
+      get root_path
+      must_respond_with :success
+    end
+  end
+
+  describe "index" do
+    it "returns a list of search recipes results" do
+      query = "rabbit"
+      VCR.use_cassette("recipes") do
+        RecipeSearchWrapper.search_recipes(query)
+
+        get query_recipe_path(query: query)
+        must_respond_with :success
+      end
+    end
+
+    it "doesn't return results for a nonsense search term" do
+      query = "zzzzzzadsfasdfasdfasdf"
+      VCR.use_cassette("recipes") do
+        RecipeSearchWrapper.search_recipes(query)
+
+        get query_recipe_path(query: query)
+        must_redirect_to root_path
+      end
+    end
+  end
```

Comment:
It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?